### ISSUE TITLE
Native Windows Support

### ISSUE THREAD

**Issue by steskalja** [beeper/bridge-manager#24]
Are there any plans for native Windows support? This is a missed opportunity, and I would love to know why this hasn't been done.

**Comment by tulir** [comment]
No, WSL exists so there's no reason to support it natively. Setting up windows CI for all the bridges would be way too much effort for a few users. It might happen if we eventually decide to bundle self-hosting bridges in beeper desktop, but otherwise very unlikely.

**Comment by 2HR3Y** [comment]
I'm interested in the possibility of native Windows support for Beeper and would love to learn more and possibly might help to build this feature. @steskalja @tulir
